docs/reference/objects/company.md

```diff
@@ -116,3 +116,10 @@ string
 {: .label .fs-1 }
 
 The Twitter username set within **My Site** > **Social**.
+
+## `company.booking_fee_name`
+{: .d-inline-block }
+string
+{: .label .fs-1 }
+
+The custom booking fee name, which can be overridden by admin, theme developers within **Easol admin** > **Company edit**.
```

Comment:
Nit: These are customer facing so perhaps we shouldn't mention permissions / where it can be overridden but just suggest getting in touch to change it?

Reply:
Good shout have added